ui/imodel-components-react/src/imodel-components-react/navigationaids/CubeNavigationAid.tsx

```diff
@@ -100,6 +100,7 @@ export interface CubeNavigationAidProps extends CommonProps {
   onAnimationEnd?: () => void;
   /** @internal */
   animationTime?: number;
+  favorHeadsUpRotation?: boolean;
```

Comment:
Should this be exposed in some way to the `CubeNavigationAidControl` or users already directly use the react component only ?

Reply:
I think we can wait until someone asks for it. I left the old logic in with a prop in case someone is relying on it, but no reason to propagate that unless it's required.